ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -276,48 +286,65 @@ def tearDown(self):
         return
 
     def runTest(self):
-        print
         thr = threading.Thread(target=self.dut_thr, kwargs={'q_from': self.q_to_dut, 'q_to': self.q_from_dut})
         thr.setDaemon(True)
         thr.start()
 
-        uptime_before = self.req_dut('uptime')
-        if uptime_before.startswith('error'):
-            self.log("DUT returned error for first uptime request")
-            self.req_dut('quit')
-            self.assertTrue(False, "DUT returned error for first uptime request")
+        # test warm reboot arp basic
+        self.test_wrarp_basic()
```

Comment:
@jcaiMR should we consider to have separate test cases in the test_wr_arp.py? It would be easier to track the different failure for basic and advance cases? Otherwise, the basic cases succussed, advanced failed, it would make the test_wr_arp failed.

Reply:
Applied in the latest diff